components/dmesg/config.go

```diff
@@ -45,25 +50,84 @@ func DmesgExists() bool {
 	return p != ""
 }
 
-const DefaultDmesgFile = "/var/log/dmesg"
+const (
+	// DefaultDmesgFile default path with dmesg file
+	DefaultDmesgFile = "/var/log/dmesg"
+	// DefaultDmesgCmd DefaultDmesgCmdWithSince default scan dmesg command (in newer util-linux it works, but older is not)
+	// some old dmesg versions don't support --since, thus fall back to the one without --since and tail the last 200 lines
+	// ref. https://github.com/leptonai/gpud/issues/32
+	DefaultDmesgCmd          = "dmesg --time-format=iso --nopager --buffer-size 163920"
+	DefaultDmesgCmdWithSince = "dmesg --time-format=iso --nopager --buffer-size 163920 --since '1 hour ago'"
+	DefaultScanDmesgCmd      = DefaultDmesgCmdWithSince + " || " + DefaultDmesgCmd + " | tail -n 200"
+	dmesgMinSupportVersion   = 2.37
```

Comment:
```suggestion

         // dmesg --version that supports "--since" flag
         // ref. https://github.com/util-linux/util-linux/blob/master/Documentation/releases/v2.37-ReleaseNotes
	dmesgSinceFlagSupportVersion = 2.37
```